googlemock/include/gmock/gmock-actions.h

```diff
@@ -717,13 +713,9 @@ class ReturnAction {
    private:
     bool performed_;
     const std::shared_ptr<R> wrapper_;
-
-    GTEST_DISALLOW_ASSIGN_(Impl);
   };
 
   const std::shared_ptr<R> value_;
-
-  GTEST_DISALLOW_ASSIGN_(ReturnAction);
```

Comment:
If there is a const member, we should still explicitly write
`Impl& operator=(const Impl&) = delete;`
Regarding move ctor, even though it invokes the copy ctor, I think it's more clear to write
`Impl(Impl&&) = default;`, otherwise it would not be implicitly declared due to the user-declared copy assignment operator.
If the `const` is removed some day, we will automatically get the optimization.